pkg/ps/ps.go

```diff
@@ -1,12 +1,16 @@
 package ps
 
 import (
+	"os"
 	"path/filepath"
+	"regexp"
 	"sort"
 	"strconv"
+	"strings"
 	"time"
 
-	"github.com/containers/libpod/cmd/podman/shared"
+	define2 "github.com/containers/libpod/pkg/ps/define"
```

Comment:
```suggestion
	psDefine "github.com/containers/libpod/pkg/ps/define"
```
Is clearer than define2.